WPFHospitalEditor/ScheduleWindow.xaml.cs

```diff
@@ -24,25 +23,23 @@ public ScheduleWindow(RecommendationDto recommendationDto)
         {
             InitializeComponent();
             this.recommendationDto = recommendationDto;
-            setAppointmentInfoContent();
-            allPatients = patientServerController.GetAllPatients().ToList();
-            foreach (Patient p in allPatients)
-                patients.Items.Add(p.Id.ToString() + " " + p.Person.Name + " " + p.Person.Surname + " - " + p.Person.Jmbg);
+            SetAppointmentInfoContent();
+            SetPatientNameNameComboBox();
         }
 
         private void Close_Click(object sender, RoutedEventArgs e)
         {
             this.Close();
         }
 
-        private void scheduleBtn_Click(object sender, RoutedEventArgs e)
+        private void ScheduleBtn_Click(object sender, RoutedEventArgs e)
         {
-            if (patients.SelectedIndex == -1)
+            if (patientsComboBox.SelectedIndex == -1)
             {
                 MessageBox.Show("You must select a patient!");
             }
             else { 
-                String patient = patients.SelectedItem.ToString();
+                String patient = patientsComboBox.SelectedItem.ToString();
                 int patientID = int.Parse(patient.Split(" ")[0]);
```

Comment:
If I understand right, you need to check if selected index is 0, because with new changes that is selected by default.

Reply:
Because of changes from the last comment, now index 0 is "None".